django/template/loader_tags.py

```diff
@@ -171,9 +171,16 @@ def render(self, context):
             # If not, try the cache and get_template().
             template_name = template
             cache = context.render_context.dicts[0].setdefault(self, {})
-            template = cache.get(template_name)
+            try:
+                template = cache.get(template_name)
+            except Exception:
+                template = None
```

Comment:
I don't think `except Exception` is appropriate here - it's too broad. This should probably be under a type check

Reply:
Ok, I will change it.